### ISSUE TITLE
Question for custom dataset

### ISSUE THREAD

**Issue by luigifaticoso** [j96w/DenseFusion#189]
Hello, 
I tried to use some real data for my dataset and attempted to use the LabelFusionel dataset tool but didn't manage to install all the dependencies because of a problem and another. 
Is there any recent way to label real RGBD data using an intel real sense camera? 
Thank you

**Comment by nihar0602** [comment]
Hi @luigifaticoso 
were you able to figure this out?

**Comment by luigifaticoso** [comment]
@nihar0602 yes, I have investigated more and found this tool https://github.com/F2Wang/ObjectDatasetTools useful for my case. Hope this can help you out.

**Comment by nihar0602** [comment]
Thank you for the response. 
I tried using that but was not able to create a proper 3D part on that. 
Can you elaborate on how you created 3D CAD/OBJ?MESH File and how did you create the dataset? 
Was it only synthetic or was it having real images? 
How did you created ground truth and passed into the network?

**Comment by luigifaticoso** [comment]
> Thank you for the response.

I tried using that but was not able to create a proper 3D part on that.

> Can you elaborate on how you created 3D CAD/OBJ?MESH File and how did you create the dataset?

I already had my model in 3D, but only mesh, I have then sampled some points on it to create the pointcloud using cloud compare. After that, I created the 3D scene using ObjectDatasetTools and aligned my pointcloud to the corresponding one in the reconstructed scene and used that to obtain the pose matrix.

> Was it only synthetic or was it having real images?

I used both synthetic and real images.

> How did you created ground truth and passed into the network?

By using ObjectDatasetTools and https://github.com/ethnhe/raster_triangle 

I know it takes a bit to get it right, for me the whole process of collecting data and data cleaning until the model took me more than 1 month.
